rel/env.sh.eex

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Sets and enables heart (recommended only in daemon mode)
+# case $RELEASE_COMMAND in
+#   daemon*)
+#     HEART_COMMAND="$RELEASE_ROOT/bin/$RELEASE_NAME $RELEASE_COMMAND"
+#     export HEART_COMMAND
+#     export ELIXIR_ERL_OPTIONS="-heart"
+#     ;;
+#   *)
+#     ;;
+# esac
+
+# Set the release to work across nodes. If using the long name format like
+# the one below (my_app@127.0.0.1), you need to also uncomment the
+# RELEASE_DISTRIBUTION variable below. Must be "sname", "name" or "none".
+export RELEASE_DISTRIBUTION=name
+export RELEASE_NODE=<%= @release.name %>@127.0.0.1
+export DATABASE_URL="ecto://postgres:${GAPP_PASS}@${GAPP_DB}:5432/mind_the_gapp_2"
+export SECRET_KEY_BASE="${GAPP_PASS}"
```

Comment:
This is probably ok for this case, but `SECRET_KEY_BASE` should probably change independently of the database password? In a typical web application, you wouldn't want the `SECRET_KEY_BASE` to be forced to change if you rotated your DB password.

For example, if your application had authentication, changing the secret key would mean that it would expire any active user sessions.